app/models/page_sequence.rb

```diff
@@ -45,6 +45,7 @@ def initialize(claim, current_slug)
 
   def slugs
     SLUGS.dup.tap do |sequence|
+      sequence.delete("where-teaching") if claim.eligibility.claim_school.present? && claim.eligibility.claim_school.closed?
```

Comment:
Could this just be `claim.eligibility.claim_school&.closed?` or is `present?` doing something special here?

Reply:
Consider the case where `claim_school == nil`. `claim.eligibility.claim_school&.closed?` would be falsy, and we would delete `where-teaching`, which I think is incorrect.

Reply:
Yep, makes sense.

Reply:
Actually I think I'm wrong. I'll check.